docs/production/security-model.md

```diff
@@ -275,6 +275,25 @@ strength allowed is controlled by two settings in
 [smokescreen-setup]: deployment.md#customizing-the-outgoing-http-proxy
 [proxy.enable_for_camo]: deployment.md#enable_for_camo
 
+## Rate limiting
```

Comment:
I didn't know which file this best belongs in, figured `security-model.md` might fit the best for a sysadmin-facing doc (else i'd put it in `subsystems` but that's more developer-facing)